templates/single.twig

```diff
@@ -46,7 +46,7 @@
                 </div>
                 <h1 class="page-header-title">{{ post.title|raw }}</h1>
                 <div class="single-post-meta">
-                    {% if not post.get_author_override and post.author.avatar %}
+                    {% if post.author.avatar %}
```

Comment:
This will fix the missing default avatar for author override posts. [eg.](https://www-dev.greenpeace.org/test-proteus/story/1467/lorem-ipsum/)